cmd/aries-js-worker/aries.js

```diff
@@ -4,18 +4,15 @@ Copyright SecureKey Technologies Inc. All Rights Reserved.
 SPDX-License-Identifier: Apache-2.0
 */
 
+'use strict'
+
+const _isNodeJS = typeof module !== 'undefined' && module.exports
```

Comment:
is this variable name global to the containing page?

Reply:
done